core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1296,6 +1296,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_BACKUP_STATE_LOCK_SAFE_MODE =
+      new Builder(Name.MASTER_BACKUP_STATE_LOCK_SAFE_MODE)
+          .setDefaultValue(false)
+          .setDescription("Ignores client state-lock requests for backup to always take the lock.")
```

Comment:
I don't know how this should be used by users.

Reply:
The typical scenario is when you can't take backups (shell/daily) due to a never-ending RPC. Restarting Alluxio master with this will guarantee that backup can take the lock and continue.

Reply:
@ggezer 
Yes, but it will never service RPCs. That will make this master unusable, right? Why would a user want an unusable master?

Reply:
The point is not "wanting an unusable master", It's "wanting a backup".
@calvinjia @ns1123 WDYT about this mode?

Reply:
I'd rather instead of being a server side property, that his is an option on the `fsadmin backup` command. That way a backup can be forced by an admin in a dire situation.

Reply:
@ns1123 This PR already makes the changes for doing the best we can in taking the backup. This particular property is introducing a new maintenance mode under which backup will be "guaranteed".